armi/materials/material.py

```diff
@@ -582,13 +582,14 @@ def isBeyondIncubationDose(self, totalDPA):
 
         """
 
-        if not self.modelConst["Rincu"]:
+        if not "Rincu" in self.modelConst:
             msg = "Material missing incubation dose"
             runLog.warning(msg, single=True, label="Missing incubation dose")
```

Comment:
This will implicitly return `None`. This is not specified in the docstring. Either we should state that this possibly returns a None (in which case we should add a type annotation ` -> Optional[bool]`), or we explicitly return some default value (probably `False`?)

Reply:
Good point. The docstring says "Passes if the material does not have an incubation dose assigned..." and in this case "passing" the isBeyondIncuatbionDose test would be getting a `False` back.